waffle-mock-contract/test/direct.test.ts

```diff
@@ -13,27 +13,27 @@ describe('Mock Contract - Integration (called directly)', () => {
   const [wallet] = new MockProvider().getWallets();
 
   it('throws readable error if mock was not set up for a method', async () => {
-    const mockCounter = await deployMockContract(wallet, Counter.interface);
+    const mockCounter = await deployMockContract(wallet, Counter.abi);
```

Comment:
@sz-piotr Hmm, is that right?

Reply:
Yeah, interface is the legacy way of accessing it. Abi has many advantages. Shorter, standard and also isn't a reserved word

Reply:
thx